extensions/librdkafka/rdkafka_utils.cpp

```diff
@@ -29,6 +29,9 @@ namespace minifi {
 namespace utils {
 
 void setKafkaConfigurationField(rd_kafka_conf_t& configuration, const std::string& field_name, const std::string& value) {
+  if (value.empty()) {
+    return;
+  }
```

Comment:
Good point, I just wanted to prevent any unnecessary config settings, but it should not be a problem when an empty value is set, removed in 98ed8e2450436c2502a71f0bc99301fa4576db2c